static/styles/app_components.css

```diff
@@ -712,3 +712,47 @@ div.overlay {
         white-space: nowrap;
     }
 }
+
+table {
```

Comment:
We'll want to do a follow-up project to replace this `table` CSS with inlining it into `table-striped` (or something), so that we're not longer resetting the CSS for tables.  It might be useful to add a comment above this indicating that.

Reply:
Done, I placed these same words above mentioned inside a comment.